src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,81 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(key, value, null);
+        Node<K, V> oldNode = table[getIndex(key)];
+        if (oldNode == null) {
+            table[getIndex(key)] = newNode;
```

Comment:
```suggestion
        int index = getIndex(key);
        Node<K, V> oldNode = table[index];
        if (oldNode == null) {
            table[index] = newNode;
```
dont call index twice